pom.xml

```diff
@@ -100,6 +100,31 @@
 				<groupId>org.springframework.boot</groupId>
 				<artifactId>spring-boot-maven-plugin</artifactId>
 			</plugin>
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-resources-plugin</artifactId>
+				<version>3.2.0</version>
+				<executions>
+					<execution>
+						<id>copy-integration-test-measures</id>
+						<phase>generate-sources</phase>
+						<goals>
+							<goal>copy-resources</goal>
+						</goals>
+						<configuration>
+							<outputDirectory>src/test/resources/FhirDataEvaluatorTest/Measures</outputDirectory>
```

Comment:
you should use the normal package structure like in `src` as path here because the same rules as in `src` to avoid collisions with other libs apply here.